app/app.html

```diff
@@ -40,7 +40,7 @@
         if (scripts.length) {
           document.write(
             scripts
-              .map(script => `<script defer src="${script}"><\/script>`)
+              .map((script) => `<script defer src="${script}"><\/script>`)
```

Comment:
why did you delete then copy same thing? just a mistake?

Reply:
eslint added parentheses around the script parameter